cccatalog-api/cccatalog/api/admin.py

```diff
@@ -5,7 +5,7 @@
 
 @admin.register(ImageReport)
 class ImageReportAdmin(admin.ModelAdmin):
-    list_display = ('reason', 'status', 'image_url', 'description')
+    list_display = ('reason', 'status', 'image_url', 'description', 'date')
```

Comment:
can we call this "created_at" instead of "date" to make it clearer what the field is for?

Reply:
Okay